packages/api/src/external/ehr/elation/shared.ts

```diff
@@ -0,0 +1,87 @@
+import { Address } from "@metriport/core/domain/address";
+import { Contact } from "@metriport/core/domain/contact";
+import { getSecretValueOrFail } from "@metriport/core/external/aws/secret-manager";
+import {
+  clientKeySchema,
+  MetriportError,
+  normalizeEmail,
+  normalizePhoneNumber,
+  normalizeUSStateForAddress,
+  normalizeZipCodeNew,
+} from "@metriport/shared";
+import { PatientResource } from "@metriport/shared/interface/external/elation/patient";
+import { getClientKeyMappingOrFail } from "../../../command/mapping/client-key";
+import { Config } from "../../../shared/config";
+import { EhrSources } from "../shared";
+
+const region = Config.getAWSRegion();
+
+export function createMetriportContacts(patient: PatientResource): Contact[] {
+  return [
+    ...patient.phones.map(p => {
+      return {
+        phone: normalizePhoneNumber(p.phone),
+      };
+    }),
+    ...patient.emails.map(e => {
+      return {
+        email: normalizeEmail(e.email),
+      };
+    }),
+  ];
+}
+
+export function createMetriportAddresses(patient: PatientResource): Address[] {
+  return [
+    {
+      addressLine1: patient.address.address_line1,
+      addressLine2:
+        patient.address.address_line2 !== "" ? patient.address.address_line2 : undefined,
```

Comment:
nit: `patient.address.address_line2 && patient.address.address_line2.trim() !== ""...`

Reply:
Added the trims, but patient.address.address_line2 is always defined, so not sure we need the undefined check.